MediaPlayer/MediaControls.js

```diff
@@ -11,15 +13,23 @@ import css from './MediaControls.module.less';
 /**
  * A set of components for controlling media playback and rendering additional components.
  *
- * @class MediaControls
+ * @class MediaControlsBase
  * @memberof agate/MediaPlayer
  * @ui
  * @private
  */
-const MediaControls = kind({
+const MediaControlsBase = kind({
 	name: 'MediaControls',
 
 	propTypes: /** @lends agate/MediaPlayer.MediaControls.prototype */ {
+		/**
+		 * `true` when the media loops.
+		 *
+		 * @type {Boolean}
```

Comment:
Please write `@default` because you define the default value of the `loop` in the `defaultProps` block.